empyrical/stats.py

```diff
@@ -1005,6 +1005,41 @@ def cagr(returns, period=DAILY, annualization=None):
     return ending_value ** (1. / no_years) - 1
 
 
+def value_at_risk(returns, cutoff):
+    return np.percentile(returns, 100 * cutoff)
+
+
+def conditional_value_at_risk(returns,
+                              weights,
+                              cutoff=0.05,
+                              lookback_days=None):
```

Comment:
As a user, when would I want to pass `lookback_days` to this vs. just slicing the dataframe myself?